src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/Utils.cs

```diff
@@ -41,5 +42,10 @@ internal static Uri AppendQueryString(Uri url, string qs)
             builder.Query = newQueryString;
             return builder.Uri;
         }
+
+        internal static bool IsRunningBlazorWasm()
```

Comment:
Let's rename this to `InRunningInBrowser` since that's the core thing that we're checking.